crates/chain/src/tx_graph.rs

```diff
@@ -1026,7 +984,7 @@ impl<A: Anchor> TxGraph<A> {
 
 /// A structure that represents changes to a [`TxGraph`].
 ///
-/// It is named "additions" because [`TxGraph`] is monotone, so transactions can only be added and
+/// It is named "changeset" because [`TxGraph`] is monotone, so transactions can only be added and
```

Comment:
This line makes a less sense with the name change. How about something like:
```suggestion
/// Because [`TxGraph`] is monotone "changeset" can only contain transactions to be added and
```